script/testing/junit/src/TracefileTest.java

```diff
@@ -149,44 +168,4 @@ public static void check(String hash1, String hash2, int n, String sql) throws E
     public static void check2(String mes) throws Exception {
```

Comment:
Likewise, rename this to be something other than `check2`. I think it should be `checkAlwaysFail`